captum/attr/_core/occlusion.py

```diff
@@ -245,8 +254,16 @@ def attribute(
         )
 
     def _construct_ablated_input(
-        self, expanded_input, input_mask, baseline, start_feature, end_feature, **kwargs
-    ):
+        self,
+        expanded_input: Tensor,
+        input_mask: Tensor,
+        baseline: Optional[
+            Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
```

Comment:
baseline here should just be Union[Tensor, int, float] (not optional), since we only apply to a single tensor.